src/prefect/cli/deployment.py

```diff
@@ -675,11 +678,19 @@ async def build(
         "--interval",
         help="An integer specifying an interval (in seconds) that will be used to set an IntervalSchedule on the deployment.",
     ),
+    interval_anchor: Optional[str] = typer.Option(
+        None, "--anchor-date", help="The anchor date for an interval schedule"
```

Comment:
do we expect an ISO-formatted string? it might be worth mentioning the expected format in this help message; I'd intuitively understand that you would expect 2022-12-24 but someone could try to do 2022/24/12

Reply:
I think it's anything that's parse-able with `pendulum.parse`, I'd have to look up the details there